src/app/banners/BannerWrapper.tsx

```diff
@@ -1,23 +1,40 @@
+import localStorageService, { STORAGE_KEYS } from 'app/core/services/local-storage.service';
+import { RootState } from 'app/store';
+import { useAppSelector } from 'app/store/hooks';
+import { PlanState } from 'app/store/slices/plan';
+import { userSelectors } from 'app/store/slices/user';
 import { useEffect, useState } from 'react';
-import Banner from './Banner';
+import { useSelector } from 'react-redux';
+import SummerBanner from './SummerBanner';
 
 const BannerWrapper = () => {
   const [showBanner, setShowBanner] = useState(false);
+  const plan = useSelector<RootState, PlanState>((state) => state.plan);
+  const isTutorialCompleted = localStorageService.get(STORAGE_KEYS.SIGN_UP_TUTORIAL_COMPLETED);
+  const userPlan = plan.subscription?.type;
+  const isNewAccount = useAppSelector(userSelectors.hasSignedToday);
 
-  const onClose = () => {
+  const onCloseBanner = () => {
     setShowBanner(false);
-    localStorage.setItem('showLifetimeBanner', 'false');
+    localStorage.setItem('showSummerBanner', 'false');
   };
 
-  useEffect(() => {
-    if (!localStorage.getItem('showLifetimeBanner')) {
-      setTimeout(() => {
+  function isBannerOpen() {
+    if (isNewAccount) {
+      if (userPlan === 'free' && isTutorialCompleted && !localStorageService.get('showSummerBanner')) {
+        setShowBanner(true);
+      }
+    } else {
+      if (userPlan === 'free' && !localStorageService.get('showSummerBanner')) {
```

Comment:
You could reduce this conditions to:
```suggestion
const shouldShowBanner = userPlan === 'free' && !localStorageService.get('showSummerBanner');

if ((isNewAccount && isTutorialCompleted && shouldShowBanner) || (!isNewAccount && shouldShowBanner)) {
  setShowBanner(true);
}
```